client/my-sites/plugins/plugin-autoupdate-toggle/index.jsx

```diff
@@ -60,7 +60,8 @@ export class PluginAutoUpdateToggle extends Component {
 
 	isAutoManaged = () =>
 		( this.props.isMarketplaceProduct && this.props.productPurchase ) ||
-		PREINSTALLED_PLUGINS.includes( this.props.plugin.slug ) ||
+		( this.props.site?.is_wpcom_atomic &&
```

Comment:
Thanks for suggesting this 👍 . I tweaked the argument to fix the issue Kraft mentioned above. Let me know what your thought about the new change.

Reply:
Thanks 👍 

Nit: I think the `isAutoManaged` function can now use some improvement, as it's pretty hard to read. Maybe it could use some comments to help with that. And/or, we could separate the condition bits into variables and that could help make the code more comprehensible.

Reply:
Thanks. I added some improvements in `isAutoManaged` as suggested. 👍